.github/workflows/build.yml

```diff
@@ -32,7 +33,9 @@ jobs:
       - name: Set up Gradle
         uses: gradle/gradle-build-action@v3
       - name: Build and Test
-        run: ./gradlew build conformanceTests demoTest --include-build ../jspecify --include-build ../checker-framework
+        run: ./gradlew build conformanceTests demoTest --include-build ../jspecify
```

Comment:
The CI results suggest that we'll need a similar change for Run Samples Tests below.

Reply:
Sorry, I had missed that.